src/index.help.ts

```diff
@@ -17,7 +16,21 @@ import {
   removeTheme,
   shopifyAuth,
 } from "./shopify";
-const UNIQUE_HIDDEN_COMMENT_STRING = "Comment created by GitHub Action `Shopify Theme Actions`";
+
+export const UNIQUE_HIDDEN_COMMENT_STRING = (SHOPIFY_AUTH: shopifyAuth): string =>
+  `Shopify Theme Actions for :${SHOPIFY_AUTH.storeUrl}`;
+
+export const getThemeName = (): string => {
+  let branch: string;
+  if (process.env.GITHUB_EVENT_NAME === "pull_request") {
+    branch = process.env.GITHUB_HEAD_REF!;
+  } else {
+    branch = process.env.GITHUB_REF_NAME!;
+  }
+  const branchParts = branch.split("/");
+  branch = branchParts.pop()!;
+  return branch;
```

Comment:
This should still include the PR number if it is from a PR i.e. `PR ${pullRequestNumber}`